src/main/java/frc/team2412/robot/commands/intake/IndexStopCommand.java

```diff
@@ -0,0 +1,18 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+
+public class IndexStopCommand extends Command {
+	private final IntakeSubsystem intakeSubsystem;
+
+	public IndexStopCommand(IntakeSubsystem intakeSubsystem) {
+		this.intakeSubsystem = intakeSubsystem;
+		addRequirements(intakeSubsystem);
+	}
+
+	@Override
+	public void initialize() {
+		intakeSubsystem.indexOut();
+	}
```

Comment:
Should be indexStop() ?

Reply:
? curious why this was resolved.

Reply:
my fault, i thought i was going to finish addressing changes at the end of last meeting